packages/shared/src/components/search/SearchHistory.tsx

```diff
@@ -40,6 +40,7 @@ export function SearchHistory({
       <span className="font-bold text-text-quaternary typo-footnote">
         {title}
       </span>
+      <h1 className="hidden">Search History</h1>
```

Comment:
Dont bots ignore hidden text? Pretty sure google bot does from what I remember.

Reply:
Yeah, you're right :/ back to the drawing board...

Reply:
I decided to implement it in a different way. For feeds, the active item in sidebar is shown as h1. For some other components, the active tab is displayed as h1.

This should get rid of all of the `<h1 className="hidden"` cases.

Reply:
Not the biggest fan tbh.
since these are only behind auth maybe we should just omit the h1 there?

It won't really do anything anyway.

Reply:
That's a valid point. The search itself is not behind auth at the moment though. Should we treat it the same? 🤔

Reply:
OK, removed most of my changes. Kept the one for search, until we decide what to do with it, since now you can go to `/search` even if not authenticated.

Reply:
I'm going to merge this PR, we can decide what to do with the search path in a followup.